slicebase/etc/mlab/slicectrl-functions

```diff
@@ -45,12 +46,12 @@ function stop_after_everything () {
 function kill_remaining () {
     local signal=$1
     KILLED=
-    pgrep -v -f "slicectrl|rsync|rpm" > /var/run/$SLICENAME/pids.afterstop
+    pgrep -v -f "slicectrl|rsync|rpm|bash|login|/bin/su" > /var/run/$SLICENAME/pids.afterstop
     if [ -f /var/run/$SLICENAME/pids.beforestart ] ; then
         for pid in `comm -23 <( sort /var/run/$SLICENAME/pids.afterstop ) \
-                             <( sort /var/run/$SLICENAME/pids.beforestart )`; do 
-            echo "STILL RUNNING: " `ps ax | grep $pid`
-            kill -$signal $pid
+                             <( sort /var/run/$SLICENAME/pids.beforestart )`; do
+            echo "STILL RUNNING: " `ps ax | grep $pid | grep -v grep` 1>&2
```

Comment:
Lol, I did not actually know that `ps $pid` worked that way.

The redirect to stderr is to send the information to the console. Previously there was actually a bug since kill_remaining is called in a subshell like this: `$( kill_remaining TERM )`. So the output from `ps` was not displayed.